client/reader/following/main.jsx

```diff
@@ -8,10 +8,8 @@ import React from 'react';
  */
 import Stream from 'reader/stream';
 
-const FollowingStream = ( props ) => {
-	return (
-		<Stream { ...props } />
-	);
-};
+const FollowingStream = props => (
+	<Stream { ...props } />
+);
```

Comment:
Hah.

Apparently I only saw so much potential for optimization, but you're obviously right :-D

Raises the question -- why not drop this file altogether, and directly import from `reader/stream`? @blowery @bluefuton